starter_templates/go/code/app/main.go

```
@@ -0,0 +1,88 @@
+				fen, err = chess.FEN(fenStr)
+				if err != nil {
+					http.Error(w, err.Error(), http.StatusBadRequest)
+					return
+				}
+			}
+
+			game := chess.NewGame(fen)
+			moves := game.ValidMoves()
+
+			w.WriteHeader(http.StatusOK)
+			w.Write([]byte(moves[0].String()))
```

golangci-lint (1.64.8)

Error return value of `w.Write` is not checked

(errcheck)